crates/jstz_cli/src/deploy.rs

```diff
@@ -16,6 +16,9 @@ pub async fn exec(
     name: Option<String>,
     network: Option<NetworkName>,
 ) -> Result<()> {
+    // maximum size of code until the DAL is implemented
+    const MAX_CODE_LENGTH: usize = 3915;
```

Comment:
How was this derived?

Also shouldn't this be a top-level constant?

Reply:
It was literally derived by trial and error. 
It doesn't actually do anything too bad if this check is omitted. It just has a less helpful error message. ("Failed to post operation")
So this change is only really giving an explanation for a common user error and noting it's due to be fixed.
The DAL stuff would happen in `jstz_node` when the operation is serialised and we can decide whether to send it directly or use the DAL.
For now I wanted to keep that change as superficial and local as possible.

Reply:
It's not a top level constant because it has no meaning outside this function. (50:50 decision omp)